src/GUI/EFCorePowerTools/Handlers/ProcessLauncher.cs

```diff
@@ -133,13 +133,14 @@ private string DropFiles(string outputPath)
                 if (version.ToString(3) == "2.0.2") testVersion = "2.0.2";
                 if (version.ToString(3) == "2.0.3") testVersion = "2.0.3";
                 if (version.ToString(3) == "2.1.0") testVersion = "2.1.0";
+                if (version.ToString(3) == "2.1.2") testVersion = "2.1.2";
             }
 
             File.Copy(Path.Combine(fromDir, "efpt.exe"), Path.Combine(toDir, "efpt.exe"), true);
 
             if (!string.IsNullOrEmpty(testVersion))
             {
-                if (testVersion == "2.1.0")
+                if (testVersion == "2.1.0" || testVersion == "2.1.2")
```

Comment:
Should drop support for 2.0.2 and only support 2.0.0 and latest patch and 2.1.0 and latest patch - I have logged https://github.com/ErikEJ/EFCorePowerTools/issues/90